shell.nix

```diff
@@ -1,5 +1,18 @@
-{ callPackage, python3, ... }:
-(callPackage ./pkgs.nix { inherit python3; }).overrideAttrs (old: {
-  src = ./.;
-  propagatedBuildInputs = old.propagatedBuildInputs ++ [ python3.pkgs.pytest ];
-})
+{pkgs ? import <nixpkgs> {}}:
+pkgs.mkShell {
+  name = "taxi-dev-shell";
+
+  buildInputs = [
+    pkgs.alejandra
+    pkgs.python3
+    pkgs.python3Packages.pip
+    pkgs.makeWrapper
+    (pkgs.callPackage ./pkgs.nix {
+      lib = pkgs.lib;
+      python3 = pkgs.python3;
+      makeWrapper = pkgs.makeWrapper;
```

Comment:
What’s this for?

Reply:
Unfinished work I’ve rewritten the shell.nix to actually be a devShell. Now it imports the built taxi and pytest in the dev shell via `nix develop`